sdks/python/apache_beam/io/gcp/tests/bigquery_io_write_pipeline.py

```diff
@@ -0,0 +1,71 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+A pipeline that writes data from a SyntheticSource to a BigQuery table.
+
+Pipeline options are described in the ParDoTest file.
```

Comment:
Can you change documentation and describe how to run the test? Also `ParDoTest` is quite different test  and some pipeline options here would require more specific description e.g. `output` here is a BigQuery table id.